runtests.ps1

```diff
@@ -0,0 +1,7 @@
+# Run the basic sets of tests using powershell
+$vstest = "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\CommonExtensions\Microsoft\TestWindow\vstest.console.exe"
+
+$tests = @("NpmTests", "NodeTests", "ProfilerTests", "JSAnalysisTests", "MockVsTests", "Nodejs.Tests.UI")

```

Comment:
We should also run SharedProjectTests and add a comment that AzurePublishing.Tests.UI are being skipped.